app/assets/stylesheets/members/index.css.scss

```diff
@@ -41,6 +59,30 @@ $parallax-background-height: $parallax-window-height;
   }
 }
 
+// Mobile
+@include breakpoint($mobile) {
+  .parallax-static-content {
+    margin-top: 0;
```

Comment:
Properties should be ordered background-color, color, height, margin-top, padding-bottom, padding-top, position, z-index